src/main/java/it/gov/pagopa/pu/registry/repository/DebtPositionRegistryRepository.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.registry.repository;
+
+import it.gov.pagopa.pu.registry.model.DebtPositionRegistry;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "debtPositionRegistries", path = "debt-position-registries")
```

Comment:
is this really needed?
```suggestion
@RepositoryRestResource(path = "debt-position-registries")
```